### PR TITLE
Update tonic to 0.11 using a version range

### PR DESCRIPTION
**Pull request by kplattret**
There is a new version of `tonic` available. This updates the various `Cargo.toml` files to accept a range of versions for that crate, allowing to update as needed.

### CODE REVIEW

File: shared_proto/Cargo.toml
Comment (eopb) [inline shared_proto/Cargo.toml:1]: The versions in this file don't need range versions because they are not published

File: tests/Cargo.toml
Comment (eopb) [inline tests/Cargo.toml:1]: The same applies for these versions

**Review by eopb — CHANGES_REQUESTED** [review]
Looks good. Just a couple of places it could be a little simpler

**Review by eopb — APPROVED** [review]

**Comment by njhill** [comment]
Thank you @eopb @kplattret!

**Pull request merged**